src/main/java/com/hariharanweb/remoteappiummanager/Server.java

```diff
@@ -31,17 +31,24 @@ public static void main(String[] args) throws IOException {
         AppiumController appiumController = new AppiumController();
 
         get("/", (req, res) -> "Server is Running!!!");
-        get("/devices", deviceController.getDevices,new JsonTransformer());
-
-        path("/device/:udid", () -> {
-            get("", deviceController.getDevice, new JsonTransformer());
+        get("/devices", deviceController.getDevices, new JsonTransformer());
+
+        path("/device", () -> {
+            get("/:udid", deviceController.getDevice, new JsonTransformer());
+            path("/ios", () -> {
+                get("/simulators", deviceController.getSimulators, new JsonTransformer());
+                get("/realDevice", deviceController.getIOSDevices, new JsonTransformer());
+            });
+            path("/android", () -> {
```

Comment:
Is it not just directly `get("/android", deviceController.getAndroidDevices, new JsonTransformer());` instead of path?

Reply:
@SrinivasanTarget we may come out something to android, so kept it as path.Lets say it would be 

/android/emulator/screenrecord